guides/doc-Managing_Security_Compliance/master.adoc

```diff
@@ -0,0 +1,82 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:security-compliance:
+:context: security-compliance
+
+= {ManagingSecurityDocTitle}
+
+ifdef::foreman-deb[]
+This feature is not available for Debian/Ubuntu.
+// Because we don't package foreman_openscap for Debian
+endif::[]
+ifndef::foreman-deb[]
+
+Security compliance management is the ongoing process of defining security policies, auditing systems for compliance with those policies and resolving instances of non-compliance.
```

Comment:
Kindly asking you to move this introduction to either a snippet, or, even better, along with the heading to a "con_" file. Reason: in downstream, I don't use the "master.adoc" or "assembly_" files for various reasons.

Reply:
Fixed. Compromise, because I want to stick with the same style as in other `master.adoc`s.